utils/index.ts

```diff
@@ -1,13 +1,32 @@
-const lzString = require('lz-string');
+import { compressToEncodedURIComponent } from 'lz-string';
 
-const compressParams = ({
+import type { Widths } from '../src/Playroom/Playroom';
+
+export interface Snippet {
+  group: string;
+  name: string;
+  code: string;
+}
+
+export type Snippets = Snippet[];
```

Comment:
Is this deliberate? If so, should we add a changeset for exposing snippet types?

Reply:
All these types are already exposed in the old `index.d.ts` ([braid usage](https://github.com/seek-oss/braid-design-system/blob/eb5a8cbdc3d7ff5be9d5056784e4ba933a143848/packages/braid-design-system/src/lib/components/private/Snippets.ts#L2)). I think if you don't specify `exports` explicitly, all types in a `.d.ts` file are exported. The new bundle _does_ explicitly export though.